src/aux-common/bots/BotEvents.ts

```diff
@@ -5640,6 +5682,39 @@ export function getVoices(taskId?: string | number): GetVoicesAction {
     };
 }
 
+/**
+ * Creates an action that adds a map overlay to the given bot.
+ * @param bot
+ * @param overlayId
+ * @param options
+ * @param taskId
+ */
+export function addBotMapOverlay(
```

Comment:
@TroyceGowdy This name should be consistent with the rest. `addBotMapLayer` instead of `addBotMapOverlay`